SU-HSDO/suhumsci
# SHS-5627: Implement: Update link color in Spotlight when teal background on Colorful:Canyon

# READY FOR REVIEW

## Summary
Fixes issue on component that it's using a Hero Text Overlay component. 

## Need Review By (Date)
05/17

## Urgency
medium

## Steps to Test
1. Create a page a Colorful and Traditional site 
2. Add the following components
- A `Banner image with full overlay and text` without image and a link in the body text
- A `Banner image with full overlay and text` with an image and a link in the body text
3. Confirm there's any color contrast issue on these two components

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
